Makefile

```
@@ -77,8 +77,7 @@ unittest-main:
 .PHONY: install-mock-generators
 install-mock-generators:
 	cd ${GOPATH}; \
-    go install github.com/vektra/mockery/v2@v2.53.3; \
-    go install github.com/golang/mock/mockgen@v1.6.0;
+    go install github.com/vektra/mockery/v2@v2.53.3;
```

```suggestion
    go install github.com/vektra/mockery/v2@v2.53.5;
```

I suggest [v2.53.5](https://github.com/vektra/mockery/releases/tag/v2.53.5) because of: https://github.com/vektra/mockery/pull/1109.

---

good call out.